libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -42,6 +45,14 @@ export class BookSearchComponent implements OnInit, OnDestroy {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    const sbInstance = this.snackBar.open(
```

Comment:
It seems here that the snackbar opens at the same time that the `addToReadingList ` action is dispatched. Shouldn't the snackbar (which confirms the addition of the book in the list) appear after the above action has finished successfully (observing `confirmedAddToReadingList`)?

Reply:
@mandarini: Review addressed, please take a look.